File: src/main/java/com/p2p/lending/service/WithdrawalService.java
package com.p2p.lending.service;



import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Approveitem;
import com.p2p.lending.pojo.Withdrawal;


public interface WithdrawalService {
	Map<String, Object>  withdrawallist(String currpage, String btn, Map<String, Object> findmap);
	List<Withdrawal> selectallw();
	Withdrawal selectone(int id);
	int updwith(int gg, int wid);
	int updwiths(int gg, int wid, String wname);
	int updmoney(int txmoney, int uid);
	int intmoney(Withdrawal w, int i);
	int sumtxmoney();
	int sumdzmoney();
	int sumsxf();
}

File: src/main/java/com/p2p/lending/service/WithdrawalService.kt
package com.p2p.lending.service


import com.p2p.lending.entity.Withdrawal
import org.springframework.stereotype.Service

@Service
interface WithdrawalService {
    fun withdrawallist(currpage: String, btn: String, findmap: Map<String, Any>): Map<String, Any>
    fun selectallw(): List<Withdrawal>
    fun selectone(id: Int): Withdrawal
    fun updwith(gg: Int, wid: Int): Int
    fun updwiths(gg: Int, wid: Int, wname: String): Int
    fun updmoney(txmoney: Int, uid: Int): Int
    fun intmoney(w: Withdrawal, i: Int): Int
    fun sumtxmoney(): Int
    fun sumdzmoney(): Int
    fun sumsxf(): Int
}
